ogzaf/macro_params.py

```diff
@@ -21,85 +23,198 @@ def get_macro_params():
+    fred_data = web.DataReader(fred_variable_dict.values(), "fred", start, end)
     fred_data.rename(
-        columns=dict((y, x) for x, y in variable_dict.items()), inplace=True
+        columns=dict((y, x) for x, y in fred_variable_dict.items()),
+        inplace=True,
     )
 
-    # make sure all dollar value data are in billions
-    fred_data["Debt held by public"] = fred_data["Debt held by public"] / 1000
-
-    # Separate quartely, monthly, and annual dataseries
+    # Separate quartely, monthly, and annual FRED dataseries
     fred_data_q = (
         fred_data[
             [
-                "Debt held by public",
                 "Nominal GDP",
                 "Total gov transfer payments",
                 "Social Security payments",
                 "Gov expenditures",
                 "Gov interest payments",
-                "GDP Per Capita",
             ]
         ]
         .resample("Q")
         .mean()
     )
-    fred_data_a = fred_data[["Labor share"]]
-    fred_data_d = fred_data[["BAA Corp Bond Rates", "10 year govt bond rate"]]
+
+    # fred_data_d = fred_data[["BAA Corp Bond Rates", "10 year govt bond rate"]]
 
     # initialize a dictionary of parameters
     macro_parameters = {}
 
     # print(fred_data.loc(str(baseline_date)))
     # find initial_debt_ratio
-    macro_parameters["initial_debt_ratio"] = pd.Series(
-        fred_data_q["Debt held by public"] / fred_data_q["Nominal GDP"]
-    ).loc[baseline_date]
+    macro_parameters["initial_debt_ratio"] = (
+        pd.Series(wb_data_q["Gross PSD Gen Gov - percentage of GDP"]).loc[
+            baseline_yearquarter
+        ]
+    ) / 100
+
+    # find initial_foreign_debt_ratio
+    macro_parameters["initial_foreign_debt_ratio"] = pd.Series(
+        wb_data_q["Gross PSD USD - external creditors"]
+        / (
+            wb_data_q["Gross PSD USD - domestic creditors"]
+            + wb_data_q["Gross PSD USD - external creditors"]
+        )
+    ).loc[baseline_yearquarter]
+
+    # find zeta_D (Share of new debt issues from government that are purchased by foreigners)
+    macro_parameters["zeta_D"] = [
+        pd.Series(
+            wb_data_q["Gross PSD USD - external creditors"]
+            / (
+                wb_data_q["Gross PSD USD - domestic creditors"]
+                + wb_data_q["Gross PSD USD - external creditors"]
+            )
+        ).loc[baseline_yearquarter]
+    ]
 
     # find alpha_T
-    macro_parameters["alpha_T"] = pd.Series(
-        (
-            fred_data_q["Total gov transfer payments"]
-            - fred_data_q["Social Security payments"]
-        )
-        / fred_data_q["Nominal GDP"]
-    ).loc[baseline_date]
+    macro_parameters["alpha_T"] = [
+        pd.Series(
+            (
+                fred_data_q["Total gov transfer payments"]
+                - fred_data_q["Social Security payments"]
+            )
+            / fred_data_q["Nominal GDP"]
+        ).loc[baseline_date]
+    ]
```

Comment:
```suggestion
    macro_parameters["alpha_T"] = [[0.041761258]]  # 2020 value from GFS 
```

Reply:
@sarvonz and @rickecon : as we discussed, I hard coded values for `alpha_t` and `alpha_g`. This should let us move forward.